package/src/manifest.rs

```diff
@@ -505,11 +537,21 @@ mod tests {
             }
         }
 
-        // Here's an exception to the rule that manifest errors are caught at eval time: the contract doesn't attempt
-        // to validate urls.
+        // Here are some exceptions to the rule that manifest errors are caught at eval time.
+
+        // The contract doesn't attempt to validate urls:
```

Comment:
Nitpick: the negative wording is strange, it sounds like the contract doesn't reject URLs, and I would expect the test to check that URLs are indeed not rejected. Maybe the positive version would be clearer, like:
```suggestion
        // The contract rejects URLs:
```

Or did you mean something else?